x/emissions/keeper/keeper.go

```diff
@@ -134,6 +146,9 @@ type Keeper struct {
 	// map of (topic, worker) -> forecast[]
 	forecasts collections.Map[collections.Pair[TopicId, ActorId], types.Forecast]
 
+	// map of (topic, reputer) -> reputer loss
+	lossBundles collections.Map[collections.Pair[TopicId, Reputer], types.ReputerValueBundle]
```

Comment:
There may be a way to consolidate `lossBundles`, `topicLastReputerCommit`, and `allLossBundles`. May avoid loading unnecessary data at places by keeping them separate though, just a thought.

Reply:
They have different purposes, but we can think about improving this in the future ofc.